### PR TITLE
skip handling irrelevant files for openapi runner

### PR DESCRIPTION
**Pull request by PelegLi**
**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

## Description
skip handling irrelevant for (mostly but not only) openapi runner. added an example file that failed the runner tests before the fix

Fixes # (issue)
https://redlock.atlassian.net/browse/BCE-7158

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] New and existing tests pass locally with my changes

### CODE REVIEW

**Comment by schosterbarak** [comment]
please add a sample UT

File: checkov/common/runners/object_runner.py [hunk 1]
```diff
@@ -92,8 +92,10 @@ def run(
                     continue
 
                 # result record
-                if result_config:
+                if result_config and isinstance(result_config, (list, dict)):
```
Comment (achiar99) [inline checkov/common/runners/object_runner.py:95]: so we will not generate record for that?

**Review by achiar99 — APPROVED** [review]

File: checkov/common/runners/object_runner.py [hunk 2]
```diff
@@ -92,8 +92,10 @@ def run(
...
                     end, start = self.get_start_end_lines(end, result_config, start)
+                else:
+                    continue
```
Comment (achiar99) [inline checkov/common/runners/object_runner.py:98]: so in this case we will not generate record?

Comment (achiar99) [reply checkov/common/runners/object_runner.py]: @PelegLi maybe it will be better to try catch here instead of checking this specific case

**Review by rotemavni — APPROVED** [review]

**Pull request merged**